### DISCUSSION TITLE
Keycloak-X and oracle database support

### DISCUSSION THREAD

**Discussion by mint-lemon-eucalyptus** [keycloak/keycloak#8701] [Q&A] [answered]
Hi, 
can you please tell me how to make the Quarkus-based distribution of Keycloak working with oracle database? In wildfly-based distribution this was possible by adding JDBC-driver to the ${keycloakHome}/modules directory. 
In the documentation was oracle driver mentioned (see https://github.com/keycloak/keycloak-community/blob/main/design/keycloak.x/configuration.md), but if i attempt to configure it with **kc.sh config** an exception will be thrown:
![image](https://user-images.githubusercontent.com/6142769/140328971-4d4b8f21-0c8b-424a-b91f-938ee7413b30.png)

**Comment by pedroigor** [comment, marked as answer]
@mint-lemon-eucalyptus Coming very soon :) We are working now on it. See https://issues.redhat.com/browse/KEYCLOAK-19310.

**Reply by pedroigor** [reply]
@mint-lemon-eucalyptus Initially, we are going to be providing support for whatever database Quarkus supports. Oracle is one of them.

**Comment by mint-lemon-eucalyptus** [comment]
@pedroigor , thank you for quick response!
